packages/perseus/src/util/fix-passage-refs.ts

```diff
@@ -101,8 +103,8 @@ const fixRendererPassageRefs = (options: any) => {
     return traverse(options, null, fixRadioWidget, fixWholeOptions);
 };
 
-const FixPassageRefs = (itemData: any): any => {
-    if (itemData._multi) {
+const fixPassageRefs = (itemData: PerseusItem): any => {
+    if ("_multi" in itemData) {
```

Comment:
If `_multi` could be in `itemData`, should the type be `PerseusItem | MultiItem`?